core/modules/installer/installer.info

```diff
@@ -0,0 +1,7 @@
+name = Installer
+description = Securely installs or updates modules, themes, and layouts via a web interface.
```

Comment:
Let's change the description to `Provides a user interface for installing and updating modules, themes, and layouts.`
* remove the word "Securely" as that's debatable ;)
* we don't use `web interface` anywhere else, let's stick to `user interface`